server.go

```diff
@@ -161,12 +161,35 @@ func (s *Server) Start() error {
 		return err
 	}
 
-	// Start the macaroon service and let it create its default macaroon in
-	// case it doesn't exist yet.
-	if err := s.startMacaroonService(true); err != nil {
+	// Create and start the macaroon service and let it create its default
+	// macaroon in case it doesn't exist yet.
+	s.macaroonService, err = lndclient.NewMacaroonService(
+		&lndclient.MacaroonServiceConfig{
+			DBPath:           s.cfg.BaseDir,
+			DBTimeout:        clientdb.DefaultPoolDBTimeout,
+			MacaroonLocation: poolMacaroonLocation,
+			MacaroonPath:     s.cfg.MacaroonPath,
+			Checkers: []macaroons.Checker{
+				macaroons.IPLockChecker,
+			},
+			RequiredPerms: RequiredPermissions,
+			DBPassword:    macDbDefaultPw,
+			LndClient:     &s.lndServices.LndServices,
+			EphemeralKey:  lndclient.MacaroonPubKey,
+			KeyLocator: &keychain.KeyLocator{
+				Family: 21,
+				Index:  0,
+			},
```

Comment:
Not sure what to use here? can we just create any constant? does it necessarily need to differ from what Lit uses?

Reply:
I'd say let's also create a key locator variable in `lndclient` that can be used in all projects. I think it makes sense to use the same shared key password for all daemons, hence the same key locator everywhere.